README.md

```diff
@@ -1 +1,226 @@
+
+
+### Run tests
+
+Open a terminal and navigate to your project's root directory.
+
+Run the following command to run ESLint and check the quality of your HTML code:
+
+
+> npx eslint **/*.html
+
+Run the following command to run stylelint and check the quality of your CSS code:
+
+
+> npx stylelint "**/*.css"
+
+These commands will run the linters and tell you about errors and warnings in your HTML and CSS code.
+
+
+
+AUTHORS
+
+## 👥 Authors: GhostEsso
+
+- GitHub: [@githubhandle](https://github.com/GhostEsso)
+- Twitter: [@twitterhandle](https://twitter.com/TambanaEssohana)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/essohanam-tambana-62aa081a1/?originalSubdomain=tg)
+
+
+
+CONTRIBUTING
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions are welcome and will be appreciated! If you would like to contribute to this project, please follow these steps:
+
+Fork this project.
+
+Create a branch for your feature > git checkout -b feature/AmazingFeature.
+
+Commit your changes > git commit -m 'Add some AmazingFeature'.
+
+Push your branch > git push origin feature/AmazingFeature.
+
+Open a pull request.
+
+Once your pull request has been reviewed and accepted, your feature will be added to the project.
+
+Feel free to contact me if you have any questions or comments on how to contribute to this project.
+
+
+SUPPORT
+
+## ⭐️ Show your support
+If you liked this project, don't hesitate to support it by giving a ⭐️!
+
+It means a lot to me and encourages me to continue working on this project and to create other interesting projects in the future.
+
+If you have any comments or suggestions to improve this project, do not hesitate to contact me.
+
+If you like this project...
+
+
+
+ACKNOWLEDGEMENTS <a name="acknowledgements"></a>
+
+## 🙏 Acknowledgments 
+
+I would like to thank NTIHINDUKA ALPHA for his excellent work in revising my README file. His feedback greatly improved the quality and clarity of my project documentation.
+
+His contribution and support have been of great help to me throughout the development of this project. I am grateful for his time and expertise.
+
+Thanks again, NTIHINDUKA ALPHA!
+
+I am grateful for any input and support for this project. If you have contributed to the project and are not on this list, please contact me so I can add you.
+
+
+
+
+## 📝 License 
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] You have added a LICENSE.md file in your root folder. Kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE
https://choosealicense.com/licenses/mit/

Your MIT link leads to a broken 404 page as shown below. 
![image](https://user-images.githubusercontent.com/84156714/219046417-a2a75fc5-12bb-4989-b1fe-0cd72672aa3b.png)